src/manifest.ts

```diff
@@ -462,6 +462,7 @@ export class Manifest {
         sha: commit.sha,
         message: commit.message,
         files: commit.files,
+        pullRequest: commit.pullRequest,
```

Comment:
Is this just the `pullRequest` #, curious why this was an optional parameter?

Reply:
This is the full pull request data. It's optional because not all commits on a branch have an associated pull request.